copier/cli.py

```diff
@@ -172,12 +222,13 @@ class CopierUpdateSubApp(cli.Application):
     generated since the last `copier` execution. To disable that, use `--no-diff`.
     """
 
-    only_diff = cli.Flag(
+    only_diff: cli.Flag = cli.Flag(
```

Comment:
This type seems to not be detected properly:

![2020-07-24_08-13](https://user-images.githubusercontent.com/973709/88365219-9c02e000-cd85-11ea-8136-3d40f32739ff.png)

Reply:
Indeed, my intuition is that the `@CopierApp.subcommand("update")` decorator has some responsibility in this :thinking:

Reply:
It's not something critical, but maybe we should open an issue upstream... Do you feel like doing that?

Reply:
Sure, issue opened here: https://github.com/pawamoy/pytkdocs/issues/60!